test/Microsoft.Azure.Devices.Edge.Test.Common/certs/LeafCertificates.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common.Certs
+{
+    public class LeafCertificates
```

Comment:
```suggestion
    public class LeafDeviceCertificates
```

Reply:
...which means we'd also need to rename `EdgeCertificates` to `EdgeDeviceCertificates`. How important is this to you? Sounds like you'd prefer the longer name and I'd prefer the shorter, but is one clearly better than the other?

Reply:
Leave it as it, not a big deal. Yup. I like more specific.